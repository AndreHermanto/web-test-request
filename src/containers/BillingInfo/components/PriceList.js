import React from 'react';
import { Row, Col } from 'react-bootstrap';
import styled from 'styled-components';

const Box = {
  border: '1px solid #eee',
  'border-bottom':'0px',
  'padding-top': '5px',
  'padding-bottom': '5px'
}
const InfoBox = styled.div`
  ${Box}
`;

const PriceBox = styled.div`
  ${Box}
`;

/**
 * PriceList - UI for input billing details.
 */
function PriceList (props) {
    return (
      <Row>
        <Col md={12}>
          <InfoBox className="col-md-9" style={props.style}>
            {props.priceList.info} {props.index}
          </InfoBox>
          <PriceBox className="col-md-2" style={props.style}>
            ${props.priceList.price}
          </PriceBox>
        </Col>
      </Row>
    );
}

export default PriceList;