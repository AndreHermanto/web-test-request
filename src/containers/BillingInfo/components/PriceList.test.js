import React from 'react';
import renderer from 'react-test-renderer';
import PriceList from './PriceList';
test('PriceList renders correctly', () => {
  const priceList =  {
    info: 'aa',
    price:111
  }
  const tree = renderer.create(<PriceList
    style={null} priceList={priceList}
     />).toJSON();
  expect(tree).toMatchSnapshot();
});