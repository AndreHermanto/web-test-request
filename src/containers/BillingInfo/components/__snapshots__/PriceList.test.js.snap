exports[`test PriceList renders correctly 1`] = `
<div
  className="row">
  <div
    className="col-md-12">
    <div
      className="col-md-9 BtjpU"
      style={null}>
      aa
       
    </div>
    <div
      className="col-md-2 dpLQwy"
      style={null}>
      $
      111
    </div>
  </div>
</div>
`;
